AzureTable/Microsoft.DataTransfer.AzureTable.FunctionalTests/packages.config

```diff
@@ -1,16 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Castle.Core" version="3.3.3" targetFramework="net451" />
-  <package id="Microsoft.Azure.CosmosDB.Table" version="0.9.0-preview" targetFramework="net451" />
```

Comment:
Should `targetFramework` say `net452`? I think NuGet extension for VS should do the right thing on its own, if you do the version bump first and then add/update a package.

Reply:
agree. since the .net version was manually updated, I updated all the versions of packages in packages.config